ui/app/common/pnc-client/rsql/comparator.js

```diff
@@ -71,6 +71,13 @@
           return ctx.next();
         };
 
+        that.notlike = function (value) {
+          value = value.replace(/\*/g, '%');
```

Comment:
Just a thought these replace lines should have been removed in 2.0 ... the rest API should now accept '*' and '?' natively. That said they could be removed in a different issue

Reply:
Got it, I will create another jira and fix it in next sprint.